client/my-sites/plans-features-main/test/plan-type-selector.jsx

```diff
@@ -46,68 +64,45 @@ describe( '<CustomerTypeToggle />', () => {
 		withWPPlanTabs: true,
 	};
 
-	beforeEach( () => {
-		global.document = {
-			location: {
-				search: '',
-			},
-		};
-	} );
+	afterEach( () => {} );
 
 	test( "Should select personal tab when it's requested", () => {
 		const props = { ...myProps, customerType: 'personal' };
-		const comp = shallow( <CustomerTypeToggle { ...props } /> );
-		expect( comp.find( 'SegmentedControl' ).length ).toBe( 1 );
-		expect(
-			comp.find( 'SegmentedControlItem[path="?customerType=personal&plan=free_plan"]' ).length
-		).toBe( 1 );
-		expect(
-			comp.find( 'SegmentedControlItem[path="?customerType=personal&plan=free_plan"]' ).props()
-				.selected
-		).toBe( true );
-		expect(
-			comp.find( 'SegmentedControlItem[path="?customerType=business&plan=free_plan"]' ).length
-		).toBe( 1 );
-		expect(
-			comp.find( 'SegmentedControlItem[path="?customerType=business&plan=free_plan"]' ).props()
-				.selected
-		).toBe( false );
+		render( <CustomerTypeToggle { ...props } /> );
+
+		const radios = screen.getAllByRole( 'radio' );
+
+		expect( radios[ 0 ] ).toHaveAttribute( 'href', '?customerType=personal&plan=free_plan' );
```

Comment:
Where we directly access `radios[0]` and `radios[1]`, is it useful to also have an assertion for the number of radio buttons found by the above query?

Reply:
Yup, done via https://github.com/Automattic/wp-calypso/pull/66116/commits/50e40cfcb47bdbaaadff38ea1e79fc6db3759e1a 👍🏻